packages/notifi-react/README.md

```diff
@@ -1,81 +1,139 @@
 
-3. manage (add/remove) targets
+- Login to Notifi Services
 
-4. Preview notification details
+-  Manage Subscriptions to Notifications
 
-### Getting Started
+- Manage Targets/Channels for Notifications
 
-1. Make sure the react environment is set up properly with the following dependencies:
+- Preview Notification Details
 
+<br/><br/>
+## Getting Started
+*Environment*
 - Node version >= 18 (with corresponding npm version)
 - React version >= 17
 
-2. Make sure you have a Notifi tenant account. we will need the following information to set up the `NotifiCardModal`.
+>If you haven't created a Notifi tenant account yet [set up an account](https://admin.notifi.network/signup?environment=prd)
 
-- tenantId
-- cardId
-
-> You can register a new tenant through [Notifi Admin Portal](https://admin.notifi.network/) for free if you do not have one yet.
-> Check out the [detail here](https://docs.notifi.network/docs/getting-started)
-
-#### Mount `NotifiCardModal` to your dapp
-
-1. Wrap your app with `NotifiContextProvider`
+<br/><br/>
+### Mount the `NotifiCardModal` to your dApp
 
+*Example*
 ```tsx
-import { NotifiFrontendClientProvider } from '@notifi-network/notifi-react';
+import { NotifiContextProvider, NotifiCardModal } from '@notifi-network/notifi-react';
+import { useEthers } from '@usedapp/core';
+import { providers } from 'ethers';
+
+const NotifiCard = () => {
+
+  const { account, library } = useEthers();
+  const signer = useMemo(() => {
+    if (library instanceof providers.JsonRpcProvider) {
+      return library.getSigner();
+    }
+    return undefined;
+  }, [library]);
+
+  if (account === undefined || signer === undefined) {
+    // account is required
+    return null;
+  }
 
-const App = () => {
-  const params = {
-    // params for contexts
-  };
   return (
-    <NotifiContextProvider {...params}>
-      <App />
-    </NotifiContextProvider>
+//tenantId/dAppId and cardId are found on the Notifi Tenant Portal. Please see Docs below.
+ <NotifiContextProvider
+    tenantId="YOUR_TENANT_ID"
+    env="Production"
+    cardId="YOUR_CARD_ID"
+    signMessage={signMessage}
+    walletBlockchain="ETHEREUM"
+    walletPublicKey={account}
+    >   
+        <NotifiCardModal darkMode={true} />
+  </NotifiContextProvider>
   );
 };
 ```
+> [Docs](https://docs.notifi.network/docs/getting-started)
+
+<br/><br/>
+
+# Advanced Customized Implementation
+
+
 
-2. Mount `NotifiCardModal` component to wherever you want in your dapp.
+The style of the `NotifiCardModal` is fully customizable. There are two approaches to customize the card style:
+<br/><br/>
 
+## Customizing the Card Styling
+
+#### **Global CSS override (*Recommended*)**
+ By overriding the default CSS classes, we can style the elements in the card.
+   [Default CSS Classes are found here.](https://github.com/notifi-network/notifi-sdk-ts/tree/stage-notifi-react-m2/packages/notifi-react/lib/style)
```

Comment:
This link is already page not found, we should locate it to the correct dir at main